recipe/meta.yaml

```diff
@@ -82,6 +82,7 @@ outputs:
         - pytest-subtests >=0.11.0
         - pytest-vcr >=1.0.2
         - pytest-xdist >=3.5.0
+        - pytorch ==2.3.1                     # [aarch64]
```

Comment:
1. This should be down below in line 188.
2. It's better to do a `!=2.4.0` for the bad version, assuming that it will be fixed in the next release.  But if a maximum pin is needed, it is better to express as `>=2.1.0,<2.4.0` than as `==2.3.1`.  Not only does that allow earlier versions that might be needed for compatibility, but it also allows later versions that might be issued as back-patched fixes.

Reply:
The bad version is 2.3.2. I had `!=2.3.2`, but I'll update it to include  >=2.1.0,!=2.3.2`